agent/agent.go

```diff
@@ -108,6 +108,11 @@ func handle(conn net.Conn, msg []byte) error {
 		}
 		time.Sleep(30 * time.Second)
 		pprof.StopCPUProfile()
+	case signal.Vitals:
+		fmt.Fprintf(conn, "goroutines: %v\n", runtime.NumGoroutine())
+		fmt.Fprintf(conn, "GOMAXPROCS: %v\n", runtime.GOMAXPROCS(0))
+		fmt.Fprintf(conn, "num CPU: %v\n", runtime.NumCPU())
+		fmt.Fprintf(conn, "OS threads: %v\n", pprof.Lookup("threadcreate").Count())
```

Comment:
Can we move this right underneath the goroutine count? Those two are more related.